Automattic/wp-calypso
# Add Jetpack Creator data

Related to #

## Proposed Changes

We implement changes in calypso to support new product, and display it on the Jetpack pricing page.

- Add jetpack creator constants
- Update tests lookups
- Add plan details (todo: adjust copy)
- Update earn upsells

## Testing Instructions



> Copy is still subject to change

1. On Jetpack Cloud live link, go to `/pricing` page
2. Ensure that at the bottom of the list, there is Creator product: 
![CleanShot 2023-10-31 at 15 30 50@2x](https://github.com/Automattic/wp-calypso/assets/8419292/3c31d0b8-39b7-45f1-999a-f34e41927d21)

3. If you click "More about Creator" link you should see this popup:
![CleanShot 2023-10-31 at 15 50 34@2x](https://github.com/Automattic/wp-calypso/assets/8419292/9ce1e07d-6bf8-4c54-95b0-937fcb59f8f1)

4. Now, on Calypso live link, go to `/checkout/jetpack/jetpack_creator_yearly`
5. You should be able to navigate through all pricing options:
![CleanShot 2023-10-31 at 15 52 00@2x](https://github.com/Automattic/wp-calypso/assets/8419292/0e0b2743-9a0a-49e6-a9c7-42c275f23bed)

6. You should also see plan features on the right side: 
![CleanShot 2023-11-02 at 13 19 03@2x](https://github.com/Automattic/wp-calypso/assets/8419292/c64892a1-9146-47e3-b855-1f2e5d408629)

7. Still on Calypso live link, go to `/earn/<your_site>`, where the site should be running only Jetpack Free
8. Make sure that upsells mention Jetpack Creator and Complete, as below:
![CleanShot 2023-10-31 at 15 54 41@2x](https://github.com/Automattic/wp-calypso/assets/8419292/e483cb61-947f-445e-9ba8-d840e7d7e6cf)

EDIT: More steps

9. Purchase Jetpack Creator monthly for the site via `/checkout/jetpack/jetpack_creator_monthly?site=<your_site>`
10. Go to `/plans/my-plan/<your_site>`
11. Jetpack Creator plan should be displayed there, together with features at the bottom of the page:
![CleanShot 2023-11-02 at 13 21 38@2x](https://github.com/Automattic/wp-calypso/assets/8419292/5ef611e6-aa9a-4a63-9d9c-a4bdcab3f99d)
![CleanShot 2023-11-02 at 13 21 23@2x](https://github.com/Automattic/wp-calypso/assets/8419292/ff760ccf-eda5-4995-9eee-6c03fdccb866)

12. Go to `/plans/<your_site>` and see the Jetpack Creator card has "Upgrade" CTA:
![CleanShot 2023-11-02 at 13 22 29@2x](https://github.com/Automattic/wp-calypso/assets/8419292/ea4dc437-7c22-4a08-91ca-f7bf06cf034c)


## Pre-merge Checklist
